nightswatcher/nightswatcher.py

```diff
@@ -107,6 +108,7 @@ def get_artifact_metadata(artifact_zip):
         commit_number = m.group("commit_number")
         ftype = m.group("ftype")
         artifact[ftype] = os.path.basename(f.strip())
+        break
```

Comment:
```suggestion
```
Hm, does Python break do more than break out of the loop? I added this more or less without thinking even though running that an extra few times hardly matters at all, but this break broke some things. Anyway, away it goes. ;-)